explainability-core/src/test/java/org/kie/trustyai/explainability/local/lime/DummyModelsLimeExplainerTest.java

```diff
@@ -54,6 +54,7 @@ void testMapOneFeatureToOutputRegression(long seed) throws Exception {
                 .get(Config.INSTANCE.getAsyncTimeout(), Config.INSTANCE.getAsyncTimeUnit());
         for (Saliency saliency : saliencyMap.getSaliencies().values()) {
             assertNotNull(saliency);
+            System.err.println(saliency);
```

Comment:
@tteofili are these `println` to stay?

Reply:
of course not, polishing commit underway :smile: